core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -32,22 +32,57 @@
  */
 public abstract class RetryingClient<I extends Request, O extends Response>
         extends SimpleDecoratingClient<I, O> {
+
+    private static final int FALLBACK_DEFAULT_MAX_RETRY_ATTEMPTS = 5;
+    private static final int DEFAULT_MAX_RETRY_ATTEMPTS;
+
+    static {
+        String value = System.getProperty("com.linecorp.armeria.defaultMaxRetryAttempts", "5");
+        if (value.matches("^-?\\d+$")) { // to check whether integer or not.
```

Comment:
How about simply doing like https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java#L105?
And I think negative value should not be accepted, right?